bot/exts/moderation/infraction/management.py

```diff
@@ -201,27 +201,21 @@ async def infraction_edit(
     # region: Search infractions
 
     @infraction_group.group(name="search", aliases=('s',), invoke_without_command=True)
-    async def infraction_search_group(self, ctx: Context, query: t.Union[UserMention, Snowflake, str]) -> None:
```

Comment:
Note that it will no longer be possible to look up deleted users by their ID. Is that a use case that needs to be supported?

Reply:
Good catch! Discussed this internally in the mod-meta channel and we decided we still want this ability, so I'll revert this change.

Reply:
Should be fixed with [this](https://github.com/python-discord/bot/pull/1742/commits/8ae21a26e4c74314d0726a37e71877f8566fe6a7) commit (also removes the `_snowflake_from_regex` function)